src/Transpiler.ts

```diff
@@ -1421,9 +1421,9 @@ export abstract class LuaTranspiler {
             const identifierName = this.transpileIdentifier(node.name);
             if (node.initializer) {
                 const value = this.transpileExpression(node.initializer);
-                return `local ${identifierName} = ${value}\n`;
+                return `local ${identifierName} = ${value};\n`;
```

Comment:
You pushed these too far down, I suggest adding the ';' in the `transpileNode` function, like you did for ExpressionStatement.

Reply:
That was my first approach, but `transpileVariableDeclaration` returns a newline, so the transpiled code would look like
```
      local a = 0
;
      foo()
```

Reply:
I think that's a bug, if removing the `\n` from transpileVariableDeclaration does not break any (non-whitespace) tests I am fine with you removing it there.

Reply:
That new line after variable declaration is definitely unintentional and should be removed.